src/configurator/v1.ml

```diff
@@ -329,8 +329,6 @@ module C_define = struct
     );
     if has_type Type.String then (
       pr {|
-#define STRINGIFY1(x) #x
-#define STRINGIFY(x) STRINGIFY1(x)
 
 #if __USE_MINGW_ANSI_STDIO && defined(__MINGW64__)
 #define REAL_ARCH_INTNAT_PRINTF_FORMAT \"ll\"
```

Comment:
by the way this is internal to ctypes, you probably don't need it.

Reply:
Thanks, I'll remove it. There will be a way to add and use your own custom macros so this use case will be covered. The end goal is to port ctypes to jbuilder, so if you see anything else that needs to be done for that happen, please let me know.